tests/sql_log_test.py

```diff
@@ -0,0 +1,21 @@
+import sys,os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+import csv
+import logic
+
+
+def write_2_log_test():
+   read_csv = csv.reader(open("tests/sql_log_test.csv",encoding = "utf8" ))
+   for row in read_csv: 
+      test_statement, answer_statement, test_status, answer_status, test_purpose, answer_purpose = row
```

Comment:
Can we try to reduce the number of variables here?

Reply:
The other way to do it, would be for us to us a list for example row[4]. The variable names seem to be easier to understand then indexing from a list